molecule/driver/azure.py

```diff
@@ -54,7 +54,7 @@ class Azure(base.Base):
         driver:
           name: azure
           ssh_connection_options:
-            -o ControlPath=~/.ansible/cp/%r@%h-%p
+            - '-o ControlPath=~/.ansible/cp/%r@%h-%p'
```

Comment:
I doubt it will work, but without space between -o and ControlPath it should. Lets see.

What I do not understand is why we did not see a failure on CI related to this?

Reply:
It's in the documentation string and not executed? Perhaps we should make sure and override a functional test with such a connection setting just to make sure?